### PR TITLE
Support for cimetrics

### PR DESCRIPTION
**Pull request by jumaffre**
Adds support for cimetrics.

All the necessary documentation describing the steps I used to set it up is available on the `cimetrics` repo: https://github.com/jumaffre/cimetrics. Feel free to contribute and improve it. In particular, the display in GitHub still needs to be improved.

Note that I've merged the two performance jobs that we had to centralise the publishing to GitHub.

### CODE REVIEW

**Comment by jumaffre** [comment]
Performance report: 
 images 
 **Have a look before merging** :bar_chart: :eyes: :thumbsup:

**Comment by jumaffre** [comment]
Performance report: 
 images 
 **Have a look before merging** :bar_chart: :eyes: :thumbsup:

**Comment by jumaffre** [comment]
Performance report: 
 images 
 **Have a look before merging** :bar_chart: :eyes: :thumbsup:

File: .vsts-ci-templates/push_perf_data.yml [hunk 1]
```diff
@@ -17,3 +17,25 @@ steps:
...
+
+# cimetrics related tasks: render and publish results for Pull Requests build
+- script: ./env.sh python -m cimetrics.plot
+  env:
+    METRICS_MONGO_CONNECTION: $(METRICS_MONGO_CONNECTION)
+  workingDirectory: build
+  displayName: 'Plot cimetrics'
+  condition: eq(variables['Build.Reason'], 'PullRequest')
+
+- script: ./env.sh python -m cimetrics.github_pr
```
Comment (achamayou) [inline .vsts-ci-templates/push_perf_data.yml]: Very nice :)

Comment (achamayou) [reply .vsts-ci-templates/push_perf_data.yml]: I wonder if we could distribute a yaml template for the steps inside the package, so it's even easier to set up?

File: tests/env.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+set -e
+
+if [ ! -f "env/bin/activate" ]
+    then
+        python3.7 -m venv env
+fi
+
+source env/bin/activate
+pip install -q -U -r ../tests/requirements.txt
+
```
Comment (achamayou) [inline tests/env.sh]: This seems identical to the existing test.sh, could we not reuse that?

**Comment by jumaffre** [comment]
Performance report: 
 images 
 **Have a look before merging** :bar_chart: :eyes: :thumbsup:

**Comment by jumaffre** [comment]
Performance report: 
 images 
 **Have a look before merging** :bar_chart: :eyes: :thumbsup:

**Comment by jumaffre** [comment]
Performance report: 
 images 
 **Have a look before merging** :bar_chart: :eyes: :thumbsup:

File: tests/env.sh (hunk 2 above)
Comment (jumaffre) [reply tests/env.sh]: As discussed, replaced it with `cimetrics_env.sh` that only installs `cimetrics`.

**Comment by cimetrics** [comment]
Performance report: 
 images 
 **Have a look before merging** :bar_chart: :eyes: :thumbsup:

**Comment by cimetrics** [comment]
Performance report: 
 images 
 **Have a look before merging** :bar_chart: :eyes: :thumbsup:

**Pull request merged**